src/cider/nrepl/middleware/test.clj

```diff
@@ -80,6 +80,19 @@
                 (str/starts-with? frame-cname class-name))))
           first))))
 
+(defn deep-sorted-maps
+  "Recursively converts all nested maps to sorted maps."
+  [m]
+  (try
+    (walk/postwalk
+     (fn [x]
+       (if (map? x)
```

Comment:
Won't it be better to check around the call site if something's map?

Reply:
Which call site do you mean? If you are referring to the call in `print-object`, we want to do a postwalk traversal to sort nested maps as well as top-level maps.

Reply:
Here is an example nested map in the test:
https://github.com/clojure-emacs/cider-nrepl/pull/917/files#diff-d6825defeb66863e777ca37cd35612c08feadc23a12cab7d9b7d7791f00d407dR285